packages/nextjs/components/JourneyCard.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image";
+import TrackedLink from "~~/components/TrackedLink";
+
+/**
+ * Site header
+ */
+export const Card = ({
+  name,
+  description,
+  src,
+  link,
+}: {
+  name: string;
+  description: string;
+  src: string;
+  link: string;
+}) => {
+  return (
+    <TrackedLink
+      id={name}
+      href={link}
+      className="flex flex-col lg:w-1/3 max-w-xs shadow-lg bg-skin rounded-[46px] mt-4"
+    >
+      <div className="flex w-full h-[200px] relative">
+        <Image src={src} fill alt="Dex Challenge" className="w-full object-cover rounded-t-[46px]" />
```

Comment:
The alt was also hardcoded in ChallengeCard component, instead we can do like in BuidlCard and set:
`alt={name}`